app/views/organizations/pets/_form.html.erb

```diff
@@ -84,6 +84,11 @@
             <%= form.hidden_field :application_paused, value: false %>
             <%= form.check_box :application_paused, { class: "form-check-input", role: "switch", id: "flexSwitchCheckChecked" }, true, false %>
         </div>
+
+        <div class="form-check form-switch">
+          <%= form.hidden_field :is_published, value: false %>
+          <%= form.check_box :is_published, { class: "form-check-input", role: "switch", required: false }, true, false %>
```

Comment:
I had to add `required:false`. I'm intrigued why the other toggle didn't require it